packages/dds/sequence/src/mapKernel.ts

```diff
@@ -53,10 +53,14 @@ interface IMapMessageHandler {
     getStashedOpLocalMetadata(op: IMapOperation): unknown;
 }
 
-interface IMapMessageLocalMetadata{
+interface IMapMessageLocalMetadata {
+    lastProcessedSeq: number
     pendingClearMessageId?: number,
     pendingMessageId?: number,
-    lastProcessedSeq: number
+    /**
+     * Metadata submitted by the handler of a ValueType operation.
+     */
+    valueMetadata?: unknown
```

Comment:
gross. is it possible to strong type this? nothing here is actually generic anymore, i get if that is hard, so not doing it now, but i would love to see this all refactored